### PR TITLE
feat: finish benchmark

### PR DESCRIPTION
**Pull request by DMwangnima**
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

**Review by jasondeng1997 — APPROVED** [review]

**Comment by jasondeng1997** [comment]
LGTM

File: tests/benchmark/kitex/server/go.mod [hunk 1]
```diff
@@ -0,0 +1,56 @@
+module github.com/kitex-contrib/codec-dubbo/tests/benchmark/kitex/server
+
+go 1.20
+
+replace github.com/apache/thrift => github.com/apache/thrift v0.13.0
+
+require (
+	github.com/apache/thrift v0.13.0
+	github.com/cloudwego/kitex v0.7.2
+	github.com/kitex-contrib/codec-dubbo v0.0.0-20230925070301-7a14362a73da
```
Comment (felix021) [inline tests/benchmark/kitex/server/go.mod:13]: replace?

File: tests/benchmark/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+stress_name="stress"
+client_name="client"
+server_name="server"
+
+if $1 = "$stress_name"; then
```
Comment (felix021) [inline tests/benchmark/deploy.sh]: not standard sh syntax? 

`if [ "$1" == "$stress_name"]; then`

File: tests/benchmark/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+elif $1 = "$client_name"; then
+  client "$2" "$3"
+elif $1 = "$server_name"; then
+  server "$2"
+else
+  echo "supporting parameter: $stress_name, $client_name, $server_name"
+  exit
+fi
+
+stress() {
```
Comment (felix021) [inline tests/benchmark/deploy.sh:68]: functions should be defined before being invoked

File: tests/benchmark/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```
Comment (felix021) [inline tests/benchmark/deploy.sh:1]: How to use this?

**Review by felix021 — APPROVED** [review]

**Pull request merged**